BothamUI/BothamEmptyCaseViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  BothamEmptyCaseViewController.swift
+//  BothamUI
+//
+//  Created by Pedro Vicente Gomez on 13/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+
+public protocol BothamEmptyCaseViewController : BothamEmptyCaseUI {
+    var currentEmptyCaseView: UIView { get }
+    var view: UIView! { get }
+}
+
+public extension BothamEmptyCaseViewController {
+    func showEmptyCase() {
+        view.addSubview(currentEmptyCaseView)
+        currentEmptyCaseView.hidden = false
+        currentEmptyCaseView.center = view.center
+    }
```

Comment:
Also would be nice in the case of tableView to set     
self.tableView.tableFooterView = [UIView new]; or hide the table view or something
Because if not, we will see the line separator.